src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -73,7 +75,9 @@ public static void configCheck(final ConfigType type) {
 			config.addDefault("Chat.Prefixes.Nonbender", "[Nonbender]");
 
 			config.addDefault("Board.Title", "&lAbilities");
-			config.addDefault("Board.SelectionPrefix", ">  &r");
+			config.addDefault("Board.Prefix.Text", "> ");
+			config.addDefault("Board.Prefix.SelectedColor", ChatColor.WHITE.toString());
+			config.addDefault("Board.Prefix.NonSelectedColor", ChatColor.DARK_GRAY.toString());
 			config.addDefault("Board.EmptySlot", "&8-- Slot {slot_number} --");
 			config.addDefault("Board.MiscSeparator", "  ------------");

```

Comment:
Could we add some default Board.Extras so that users know what the template should be like? I recommend RaiseEarthWall and SurgeWave.